webanno-curation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/curation/agreement/AgreementUtils.java

```diff
@@ -267,7 +266,7 @@ private static AgreementResult makeStudy(DiffResult aDiff, Collection<String> aU
             // - if we are looking for a primitive type and encounter a subposition
             // - if we are looking for a non-primitive type and encounter a primary position
             // this is an inverted XOR!
-            if (!(isPrimitiveFeature ^ isSubPosition)) {
+            if (isPrimitiveFeature == isSubPosition) {
```

Comment:
Ok, logically the same, but now the comment above seems out-of-place. I would tend towards keeping the inverted XOR as it is explained - or you should change the explanatory comments as well.